kymatio/scattering1d/frontend/jax_frontend.py

```diff
@@ -0,0 +1,15 @@
+from ...frontend.jax_frontend import ScatteringJax
+from .numpy_frontend import ScatteringNumPy1D
+from .base_frontend import ScatteringBase1D
+
+class ScatteringJax1D(ScatteringJax, ScatteringNumPy1D, ScatteringBase1D):
```

Comment:
Explanation:
1. Multiple inheritance generally looks for methods/attributes in the order of the classes specified above. Hence `ScatteringJax` goes first in order to have priority adding its attributes in.
2. Apart from the inheritance of `ScatteringNumpPy1D` this class and its `__init__` look exactly like the one in `ScatteringNumPy1D` itself.
3. I had to include `ScatteringBase1D` because I needed to overwrite the `__init__`, basically copying and pasting the `__init__` from `ScatteringNumPy1D` but with the backend changed. I had to do that because otherwise it was doing something numpy-specific from the `ScatteringNumPy1D.__init__`. In order to do that overwriting I needed access to `ScatteringBase1D` (from which `ScatteringNumPy1D` also inherits and which it uses in its `__init__`)
4. I initially tried switching positions 2 and 3 and I got a multiple inheritance error (because `ScatteringNumPy1D` derives from `ScatteringBase1D`) and so not all permutations are actually allowed.

Will add some comments

Reply:
added some comments

Reply:
For 3. I don't know that you need to include `ScatteringBase1D` in the inheritance list just to call its constructor. It is a super-super-class of the current one, so there's no issue calling its constructor like you're doing. I tried removing this locally and tests pass.

Reply:
@MuawizChaudhary update inheritance?